src/DwitTech.AccountService.Data/repository/IUserRepository.cs

```diff
@@ -9,7 +9,22 @@ namespace DwitTech.AccountService.Data.Repository
 {
     public interface IUserRepository
     {
-        
+        Task<ValidationCode> GetActivationDetail(string activationCode);
+        Task<bool> GetUserStatus(int id);
+        Task<bool> ValidateActivationCodeExpiry(string activationCode);
+        Task UpdateUserStatus(ValidationCode validationDetails);
+       
+        public class ValidationCode
```

Comment:
You may need to sort this out with boss to know what will happen to it, cos I don't think it should be here. It'll probably cause a merge conflict with Maryann's migrations.

Reply:
> You may need to sort this out with boss to know what will happen to it, cos I don't think it should be here. It'll probably cause a merge conflict with Maryann's migrations.

Resolved